appeals/api/src/server/documents/documents.routes.js

```diff
@@ -0,0 +1,25 @@
+import { Router as createRouter } from 'express';
+import { asyncHandler } from '../middleware/async-handler.js';
+import { getDocuments } from './documents.controller.js';
+
+/** @typedef {import('@pins/appeals.api').Schema.Folder} Folder */
+
+const router = createRouter();
+
+router.get(
+	'/:appealId',
+	/*
+		#swagger.tags = ['Appeal Documents']
+		#swagger.path = '/appeals/documents/{appealId}'
```

Comment:
Perhaps it would be better for the URL for this endpoint to be `/appeals/:appelId/documents` rather than `/appeals/documents/:appealId` as it would be more consistent with the other endpoints

Reply:
ok addressed...